resubmit.py

```diff
@@ -251,9 +209,9 @@ def extendWorkflow(workflow, user, group, verbose=False, events=None, firstlumi=
     firstlumi = int(firstlumi)
 
     # Get info about the workflow to be cloned
-    helper = reqMgrClient.retrieveSchema(workflow)
-    schema = modifySchema(helper, workflow, user, group, None, events, firstlumi, None)
-    schema['OriginalRequestName'] = workflow
+    cache = reqMgrClient.getWorkflowInfo(url, workflow)
```

Comment:
looks like RequestNumEvents in not in that reqMgrClient.getWorkflowInfo
this is making things fallback to 1000 event that is the default.